scipy/special/_basic.py

```diff
@@ -2962,7 +2962,9 @@ def stirling2(N, K, exact=True):
         Stirling numbers of the second kind count the number of ways to
         partition a set with N elements into K non-empty subsets.
 
-        This is often expressed as "N subset K".
+        The values this function returns are calculated using a dynamic
```

Comment:
I added this sentence to advertise the DP shared sub-problems approach to the solution.